README.md

```diff
@@ -4,6 +4,14 @@
 * Add AKS clusters to kubeconfig via the GUI
 * Run AppLens detector diagnostics on your AKS clusters
 
+## Telemetry
+
+This extension collects telemetry data to help us build a better experience for building applications with Kubernetes and VS Code. We only collect the following data:
+
+* Which commands are executed, and whether they are executed against an Azure cluster.
+
+We do not collect any information about image names, paths, etc. The extension respects the `telemetry.enableTelemetry` setting which you can learn more about in our [FAQ](https://code.visualstudio.com/docs/supporting/faq#_how-to-disable-telemetry-reporting).
```

Comment:
Can we add a sentence about the privacy statement like below?

"We do not collect any information about image names, paths, etc. Read our [privacy statement](https://privacy.microsoft.com/privacystatement) to learn more. If you don’t wish to send usage data to Microsoft, you can set the `telemetry.enableTelemetry` setting to `false`. Learn more in our [FAQ](https://code.visualstudio.com/docs/supporting/faq#_how-to-disable-telemetry-reporting)."

Reply:
Thank you, make sense these lines are kind of copied from here : https://github.com/Azure/vscode-kubernetes-tools#telemetry so please will let you and Ivan agree and I will make changes for consistency on the other page as well. 😊